contracts/src/v0.1/RequestResponseCoordinator.sol

```diff
@@ -223,7 +264,7 @@ contract RequestResponseCoordinator is
         return (sDirectPaymentConfig.fulfillmentFee, sDirectPaymentConfig.baseFee);
     }
 
-    function getPrepaymentAddress() external view returns (address) {
+    function getPrepaymentAddress() public view returns (address) {
```

Comment:
@KelvinThai Is there a reason to change it from `external` to `public`?

Reply:
It was a mistake. let me roll it back to external. I'll be more attention on self-code review